clients/fides-js/src/lib/gpp/constants.ts

```diff
@@ -1 +1,356 @@
+        gpp_notice_fields: [UsCoV1Field.SHARING_NOTICE],
+        gpp_mechanism_fields: [],
+      },
+      us_ct: {
+        gpp_notice_fields: [UsCtV1Field.SHARING_NOTICE],
+        gpp_mechanism_fields: [],
+      },
+    },
+    data_sales: {
+      // See question above: L34
+      //   us: {
+      //     gpp_notice_fields: [UsNatV1Field.SALE_OPT_OUT_NOTICE],
+      //     gpp_mechanism_fields: [UsNatV1Field.SALE_OPT_OUT],
+      //   },
+      us_co: {
+        gpp_notice_fields: [UsCoV1Field.SALE_OPT_OUT_NOTICE],
+        gpp_mechanism_fields: [
+          {
+            field: UsCoV1Field.SALE_OPT_OUT,
+            not_available: 0,
+            opt_out: 1,
+            not_opt_out: 2,
+          },
+        ],
+      },
+      us_ct: {
+        gpp_notice_fields: [UsCtV1Field.SALE_OPT_OUT_NOTICE],
+        gpp_mechanism_fields: [
+          {
+            field: UsCtV1Field.SALE_OPT_OUT,
+            not_available: 0,
+            opt_out: 1,
+            not_opt_out: 2,
+          },
+        ],
+      },
+      us_ut: {
+        gpp_notice_fields: [UsUtV1Field.SALE_OPT_OUT_NOTICE],
+        gpp_mechanism_fields: [
+          {
+            field: UsUtV1Field.SALE_OPT_OUT,
+            not_available: 0,
+            opt_out: 1,
+            not_opt_out: 2,
+          },
+        ],
+      },
+      us_va: {
+        gpp_notice_fields: [UsVaV1Field.SALE_OPT_OUT_NOTICE],
+        gpp_mechanism_fields: [
+          {
+            field: UsVaV1Field.SALE_OPT_OUT,
+            not_available: 0,
+            opt_out: 1,
+            not_opt_out: 2,
+          },
+        ],
+      },
+      us_ia: {
+        // DEFER: Iowa isn't part of the GPP spec yet
+        gpp_notice_fields: [],
+        gpp_mechanism_fields: [],
+      },
+    },
+    sensitive_personal_data_sharing: {
+      us: {
+        gpp_notice_fields: [
+          UsNatV1Field.SENSITIVE_DATA_LIMIT_USE_NOTICE,
+          UsNatV1Field.SENSITIVE_DATA_PROCESSING_OPT_OUT_NOTICE,
+        ],
+        gpp_mechanism_fields: [
+          {
+            field: UsNatV1Field.SENSITIVE_DATA_PROCESSING,
+            not_available: Array(12).fill(0),
+            opt_out: Array(12).fill(1),
+            not_opt_out: Array(12).fill(2),
+          },
+          {
+            field: UsNatV1Field.PERSONAL_DATA_CONSENTS,
+            not_available: 0,
+            opt_out: 1,
+            not_opt_out: 2,
+          },
+          {
+            field: UsNatV1Field.KNOWN_CHILD_SENSITIVE_DATA_CONSENTS,
+            not_available: Array(2).fill(0),
+            opt_out: Array(2).fill(1),
+            not_opt_out: Array(2).fill(2),
+          },
+        ],
+      },
+      us_ca: {
+        gpp_notice_fields: [UsCaV1Field.SENSITIVE_DATA_LIMIT_USE_NOTICE],
+        gpp_mechanism_fields: [
+          {
+            field: UsCaV1Field.SENSITIVE_DATA_PROCESSING,
+            not_available: Array(9).fill(0),
+            opt_out: Array(9).fill(1),
+            not_opt_out: Array(9).fill(2),
```

Comment:
this was what I ended up doing to handle [this question](https://ethyca.atlassian.net/wiki/spaces/EN/pages/2852945989/2023-11-30+Fides.js+GPP+implementation?focusedCommentId=2863038640) but this might not be the right assumption @mfbrown